docs/books/admin_guide/13-softwares.md

```diff
@@ -900,3 +900,121 @@ Is this ok [y/N]:
 ### Conclusion
 
 EPEL is not an official repository for RHEL. But it can be useful for administrators and developers who work with RHEL or derivatives and need some utilities prepared for RHEL from a source they can feel confident about.
+
+## DNF Plugins
+
+The `dnf-plugins-core` package adds plugins to `dnf` that will be useful for managing your repositories.
+
+!!! NOTE
+    See more informations here: https://dnf-plugins-core.readthedocs.io/en/latest/index.html
+
+Install the package on your system:
+
+```
+dnf install dnf-plugins-core
+```
+
+Not all plugins will be presented here but you can refer to the package documentation for a complete list of plugins and detailed information.
+
+### `config-manager` plugin
+
+Manage DNF options, add repos, or disable them.
+
+Examples:
+
+* Download a `.repo` file and use it:
+
+```
+dnf config-manager --add-repo https://packages._centreon_.com/rpm-standard/23.04/el8/_centreon_.repo
+```
+
+* You can also set an url as a base url for a repo:
+
+```
+dnf config-manager --add-repo https://repo.rocky.lan/repo
+```
+
+* Enable or disable one or more repos:
+
+```
+dnf config-manager --set-enabled epel centreon
+dnf config-manager --set-disabled epel centreon
+```
+
+* Add a proxy to your config file:
+
+```
+dnf config-manager --save --setopt=*.proxy=http://proxy.rocky.lan:3128/
+```
+
+### `copr` plugin
+
+`copr` is an automatic rpm forge, providing a repo with built packages.
+
+* Activate a copr repo:
+
+```
+copr enable xxxx
+```
+
+### `download` plugin
+
+Download rpm package instead of installing it:
+
+```
+dnf download ansible
+```
+
+If you just want to obtain the remote location url of the package:
+
+```
+dnf download --url ansible
+```
+
+Or if you want to also download the dependencies:
+
+```
+dnf download --resolv --alldeps ansible
+```
+
+### `needs-restarting` plugin
+
+After running an `dnf update`, the running processes will continue to run but with the old binaries. In order to take into account the code changes and especially the security updates, they have to be restarted.
```

Comment:
```suggestion
After running a `dnf update`, the running processes will continue to run but with the old binaries. In order to take into account the code changes and especially the security updates, they have to be restarted.
```

Reply:
I'd add for after a `dnf update` (or any package update really) :
`rpmconf -a -c` (not sure if this pkg is default or from EPEL)
`dnf needs-restarting -s` to check if services need restarting
`dnf needs-restarting -r` to check if a reboot may be required
`dnf needs-restarting -s -r` to do both in one run

Reply:
@NezSez:
I want to keep as simple as possible this admin guide. I think your use case for rpmconf -a -c (never needed to use it personnaly) is maybe something that should not happened? In that case, a guide would be better.

Reply:
@NezSez your descriptions for needs-restarting are better than mine.
Please start a review in the github UI and make a suggestion as Serge did just before you.
Like that, the changes will be commited and will belong to you in a git-blame

Reply:
Yes, I'm familiar with git, not so much GitHub, and I didn't have time to figure out it's interface, so I just made the comment.  I also haven't read this admin book so not familiar with context and figured you or Serge would know what to do with the info.